[ProBuilder 6] First fixes for preview (#499)
Unity-Technologies/com.unity.probuilder: CHANGELOG.md

The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).

## [Unreleased]

### Fixed

> - [PBLD-103] Fixed a bug where deleting a PolyShape while creating it would create a new GameObject.

Reviewer: ```suggestion
- [PBLD-103] Fixed a bug where creating a PolyShape, then immediately deleting it  would create a new GameObject.
```

> - [PBLD-99] Removed ProBuilder shortcut conflicting with new default shortcuts in the Editor.

Reviewer: ```suggestion
- [PBLD-99] Removed ProBuilder shortcut that conflicted with default shortcuts in the Editor.
```

> - [PBLD-98] Fixed a bug where the cut tool cursor would appear on top of overlays.

Reviewer: ```suggestion
- [PBLD-98] Fixed a bug where the cut tool cursor appeared on top of overlays.
```

> - [PBLD-95] Fixed several errors messages spamming the console on package import.

Reviewer: ```suggestion
- [PBLD-95] Fixed several errors messages that spammed the console when a package was imported.
```

> - [PBLD-93] Fixed a bug where entering the PolyShape tool with a docked inspector would throw errors.

Reviewer: ```suggestion
- [PBLD-93] Fixed a bug where activating the PolyShape tool with a docked inspector would throw errors.
```

> - Fixed a bug where creating a polyshape mesh was not properly finalizing the mesh creation. 

Reviewer: ```suggestion
- Fixed a bug where creating a PolyShape mesh was not properly finalizing the mesh creation. 
```

- Fixed a bug where assigning a material from the Material Editor was not working in GameObject mode.
- 
### Changes

- Removed the Edit Mesh button from ProBuilderMesh Inspector.

> - Added a close loop feedback with PolyShape tool and improve the handles.

Reviewer: ```suggestion
- Added a close loop feedback with PolyShape tool and improved the handles.
```

> - Added better tooltips when adding removing vertices using the PolyShape tool.

Reviewer: ```suggestion
- Improved tooltips for adding and removing vertices with the PolyShape tool.
```

> - Material editor: Changed Quick material shortcut to Ctrl + Shift + Mouse2, and add it to the shortcut manager.

Reviewer: ```suggestion
- Material editor: Changed the Quick material shortcut to **Ctrl+Shift+Mouse2**, and add this new shortcut to the Shortcut Manager.
```

- Material editor: Improved the 'Match selection' button position to make it more self-explanatory.

> - Smooth Group editor: Update the number of smoothing group to 30.

Reviewer: ```suggestion
- Smooth Group editor: Updated the number of smoothing groups to 30.
```
